packages/nextclade/src/qc/ruleSnpClusters.cpp

```diff
@@ -72,8 +87,13 @@ namespace Nextclade {
       return {};
     }
 
+    // NOTE: we exclude reversions of deletions
+    // See: https://github.com/nextstrain/nextclade/issues/707
+    const auto privateSubstitutions = eraseIntersection(query.privateNucMutations.privateSubstitutions,
+      query.privateNucMutations.reversionsOfDeletions);
+
     // TODO: should we also account for result. privateDeletions here
```

Comment:
Are private deletions counted towards the SNP cluster rule or not, at the moment? Comment sounds like they're not. If they were, it would have to be as ranges, since otherwise a simple 6nt deletion would make the rule fire.

Reply:
No, deletions are not counted (and were not counted before)